src/iOS/Xamarin.iOS/Samples/Augmented Reality/CollectDataAR/CollectDataAR.cs

```diff
@@ -0,0 +1,607 @@
+using Foundation;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using UIKit;
+
+namespace ArcGISRuntimeXamarin.Samples.CollectDataAR
+{
+    [Register("CollectDataAR")]
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Collect data in AR",
+        "Augmented reality",
+        "Tap on real-world objects to collect data.",
+        "")]
+    [ArcGISRuntime.Samples.Shared.Attributes.OfflineData()]
+    public class CollectDataAR : UIViewController
+    {
+        // UI controls.
+        private ARSceneView _arView;
+        private UILabel _helpLabel;
+        private UIBarButtonItem _calibrateButton;
+        private UIBarButtonItem _addButton;
+        private CalibrationViewController _calibrationVC;
+        private UISegmentedControl _realScalePicker;
+
+        // Track when user is changing between AR and GPS localization.
+        private bool _changingScale;
+
+        // Track whether or not the user is calibrating the AR scene view.
+        private bool _isCalibrating = false;
+
+        // Feature table for collected data about trees.
+        private ServiceFeatureTable _featureTable = new ServiceFeatureTable(new Uri("https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/AR_Tree_Survey/FeatureServer/0"));
+
+        // Elevation source for AR scene calibration.
+        private ArcGISTiledElevationSource _elevationSource;
+        private Surface _elevationSurface;
+
+        // Graphics for tapped points in the scene.
+        private GraphicsOverlay _graphicsOverlay;
+        private SimpleMarkerSceneSymbol _tappedPointSymbol = new SimpleMarkerSceneSymbol(SimpleMarkerSceneSymbolStyle.Diamond, System.Drawing.Color.Orange, 0.5, 0.5, 0.5, SceneSymbolAnchorPosition.Center);
+
+        // Location data source for AR and route tracking.
+        private AdjustableLocationDataSource _locationSource = new AdjustableLocationDataSource();
+
+        private bool IsCalibrating
+        {
+            get => _isCalibrating;
+            set
+            {
+                _isCalibrating = value;
+                if (_isCalibrating)
+                {
+                    // Show the base surface so that the user can calibrate using the base surface on top of the real world.
+                    _arView.Scene.BaseSurface.Opacity = 0.5;
+
+                    // Enable scene interaction.
+                    _arView.InteractionOptions.IsEnabled = true;
+                    ShowCalibrationPopover();
+                }
+                else
+                {
+                    // Hide the base surface.
+                    _arView.Scene.BaseSurface.Opacity = 0;
+
+                    // Disable scene interaction.
+                    _arView.InteractionOptions.IsEnabled = false;
+                    _calibrationVC.DismissViewController(true, null);
+                }
+            }
+        }
+
+        public override void LoadView()
```

Comment:
I recommend moving this further down to give it less prominence, since it is less directly interesting to the AR implementation.